README.md

```diff
@@ -200,7 +206,7 @@ Check the list of pipelines you have created using the CLI:
 ```
 $ tkn pipeline ls
 NAME             AGE              LAST RUN   STARTED   DURATION   STATUS
-build-pipeline   25 seconds ago   ---        ---       ---        ---
+deploy-pipeline  25 seconds ago   ---        ---       ---        ---
```

Comment:
This was changed since the `pipeline` used in the tutorial is actually `deploy-pipeline`